### PR TITLE
[mesheryctl] add `--reset` flag to `mesheryctl stop`

### LINKED ISSUE #850: [mesheryctl] Add `--reset` flag to `mesheryctl stop`
**Current Behavior**
Currently, `mesheryctl stop` stops all Meshery containers, removes their instances, prunes their connected volumes, and resets the meshery.yaml config back to default settings.

**Desired Behavior**
Change mesheryctl stop behavior to not reset meshery.yaml unless explicitly told to do so with an additional `--reset` flag. This flag will invoke the `cleanup` command, which should soon be renamed `reset`.

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

fixes #850

### CODE REVIEW

File: docs/pages/overview.md [hunk 1]
```diff
@@ -3,12 +3,14 @@ layout: page
...
 permalink: overview
 ---
-# Introducing Meshery 
-The multi-service mesh management plane adopting, operating and developing on different service meshes. 
-Meshery facilitates learning about functionality and performance of service meshes and incorporates the collection and display of metrics from applications running on or across service meshes. 
-Meshery provides this high-level functionality: 
-
-1. Performance Benchmarking
+# Introducing Meshery
+The service mesh management plane adopting, operating and developing on different service meshes.
```
Comment (kushthedude) [inline docs/pages/overview.md:7]: ~Adopting~ Adapting would suit better here.

File: mesheryctl/cmd/cleanup.go [hunk 2]
```diff
@@ -22,14 +22,14 @@ import (
 // cleanupCmd represents the cleanup command
 var cleanupCmd = &cobra.Command{
 	Use:   "cleanup",
-	Short: "Clean up Meshery",
-	Long:  `Clean up Meshery configuration`,
+	Short: "Cleanup Meshery's configuration",
+	Long:  `Reset Meshery to it's default configuration.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		log.Info("Cleaning Meshery config...")
+		log.Info("Reseting Meshery config...")
```
Comment (kushthedude) [inline mesheryctl/cmd/cleanup.go:28]: Resetting*

Comment (leecalcote) [reply mesheryctl/cmd/cleanup.go:28]: Ha! Good eye.

**Comment by leecalcote** [comment]
@anandrkskd yes, it doesn't hurt to provide this `--reset` flag there as well. I suppose folks are about equally as likely to use the flag in either place.

Two other items are needed in this same context:

1) a renaming of `cleanup` to `reset` as the name of the command.
2) confirmation that `--config <file>` works with `--start`. In my tests, it doesn't.

File: docs/pages/overview.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/overview.md:7]: Good comment given the pervasiveness of adapters in this project. In this case, we're trying to describe part of the initial value that people get when using the project (certainly adapters to different meshes is one of those), the first of those starting with adoption of a service mesh (facilitating understanding of how they work and selection of which one to use).

**Comment by leecalcote** [comment]
> Two other items are needed in this same context:
> 
> 1. a renaming of `cleanup` to `reset` as the name of the command.
> 2. confirmation that `--config <file>` works with `--start`. In my tests, it doesn't.

@anandrkskd would you be so kind as to file issues on these two items?

**Comment by anandrkskd** [comment]
Sure @leecalcote

**Pull request merged**